Automattic/wp-calypso
# Add/BBE generic feedback section

#### Proposed Changes

<img width="1492" alt="image" src="https://user-images.githubusercontent.com/3422709/195854062-df1f5041-2238-496c-a4ef-b613a75b142e.png">


#### Testing Instructions



* Apply D90023-code
* Go through the DIFM flow `/start/do-it-for-me` and complete a purchase and complete it
* The post purchase website-content collection page should have a feedback section
* Add some feedback and submit
* Further testing instructions available at D90023-code


#### Pre-merge Checklist






Fixes Automattic/martech#1146
